src/me/mrCookieSlime/Slimefun/listeners/BlockListener.java

```diff
@@ -155,4 +96,38 @@ public void onRightClick(PlayerInteractEvent e) {
 			}
 		}
 	}
+	
+	protected boolean compareMaterials(Block b, Material[] materials, boolean onlyTwoWay)
+	{
+		return compareMaterials(b, materials[0], materials[1], materials[2], materials[3], materials[4], materials[5], materials[6], materials[7], materials[8], onlyTwoWay);
+	}
+	
+	protected boolean compareMaterials(Block b, Material mat1, Material mat2, Material mat3, Material mat4, Material mat5, Material mat6, Material mat7, Material mat8, Material mat9, boolean onlyTwoWay)
```

Comment:
Creating a method with this many arguments is really not a good idea.
You can also just use the method and above and read the Materials directly from the array instead.